### PR TITLE
Find all applications by SSN

### PR DESCRIPTION
**Pull request by elahti**

### CODE REVIEW

File: src/clj/ataru/applications/application_store.clj [hunk 1]
```diff
@@ -204,8 +204,17 @@
 (defn get-application [application-id]
   (unwrap-application (first (exec-db :db yesql-get-application-by-id {:application_id application-id}))))
 
-(defn get-latest-application-by-key [application-key]
-  (unwrap-application (first (exec-db :db yesql-get-latest-application-by-key {:application_key application-key}))))
+(defn get-latest-application-by-key [application-key organization-oids]
+  (println (str "get-latest-application-by-key: " application-key))
```
Comment (starvinraven) [inline src/clj/ataru/applications/application_store.clj]: Use logging fns or remove the printlns (also at row 214)?

File: src/clj/ataru/applications/application_store.clj [hunk 2]
```diff
@@ -204,8 +204,17 @@
...
+  (unwrap-application (first (exec-db :db yesql-get-latest-application-by-key {:application_key              application-key
```
Comment (starvinraven) [inline src/clj/ataru/applications/application_store.clj]: Threading macro might make this a bit easier to read?

File: src/cljs/ataru/virkailija/application/view.cljs [hunk 3]
```diff
@@ -296,14 +296,23 @@
...
+        ssn                (or (-> answers :ssn :value) (-> answers :birth-date :value))
+        hakukohde-name     (-> application :tarjonta :hakukohde-name)
+        applications-count (:applications-count application)
+        koulutus-info      (koulutus/koulutukset->str (-> application :tarjonta :koulutukset))]
     [:div.application__handling-heading
-     [:h2.application-handling__review-area-main-heading (str pref-name " " last-name ", " ssn)]
+     [:div.application-handling__review-area-main-heading-container
+      [:h2.application-handling__review-area-main-heading (str pref-name " " last-name ", " ssn)]
+      (when (> applications-count 1)
+      [:a.application-handling__review-area-main-heading-applications-link
```
Comment (starvinraven) [inline src/cljs/ataru/virkailija/application/view.cljs]: The indentation seems wrong here

File: src/clj/ataru/applications/application_store.clj (hunk 1 above)
Comment (elahti) [inline src/clj/ataru/applications/application_store.clj]: Accidentally left `(println)` that was used during development to the production level code. Removed completely.

File: src/clj/ataru/applications/application_store.clj (hunk 2 above)
Comment (elahti) [inline src/clj/ataru/applications/application_store.clj]: Changed to use `->`.

File: src/cljs/ataru/virkailija/application/view.cljs (hunk 3 above)
Comment (elahti) [inline src/cljs/ataru/virkailija/application/view.cljs]: Fixed indentation.

**Pull request merged**